src/elements/radio-button/radio-button-group/radio-button-group.ts

```diff
@@ -79,7 +80,7 @@ class SbbRadioButtonGroupElement extends SbbDisabledMixin(LitElement) {
   /**
    * Size variant.
    */
```

Comment:
To improve documentation we could add `@default` which defines the default value for documentation. (this would apply to all changed components)
```suggestion
   * @default 'm' / 'xs' (lean)
   */
```